frontend/ui/quickstart.lua

```diff
@@ -46,7 +47,27 @@ div.table > div > *:nth-child(2) { text-align: left; hyphens: none; background-c
 div.table > div > *:nth-child(3) { white-space: nowrap; }
 ]]
 
-local quickstart_guide = T(_([[
+local quickstart_guide
+if Device:hasScreenKB() or Device:hasSymKey() then
+    quickstart_guide = _([[
+<div class="logo">![KOReader](resources/koreader.svg)</div>
```

Comment:
For this one (I let you look if that's needed for others), it's hard go guess what is different from the generic chunk.
Might be worth a comment:
`-- On Non-Touch, not showing "Frontlight", showing specific section "Shortcuts"`
(if that's all there is after 1 mn looking for diffs)